services/src/main/java/com/chatbot/services/ChatServiceControllers/HangoutsAuth.java

```diff
@@ -0,0 +1,48 @@
+package com.chatbot.services.ChatServiceControllers;
```

Comment:
As I mentioned earlier, directory names should be in lowercase

Reply:
updated